contracts/protocol/pool/PoolStorage.sol

```diff
@@ -48,4 +48,7 @@ contract PoolStorage {
 
   // Maximum number of active reserves there have been in the protocol. It is the upper bound of the reserves list
   uint16 internal _reservesCount;
+
+  // Mapping of addresses allowed to perform force liquidations
+  mapping(address => bool) internal _ForcedLiquidationWhitelist;
```

Comment:
I would rename it to `_forcedLiquidationWhitelist`, to be consistent with the rest of the names in the storage